charts/camunda-platform-8.8/values.yaml

```
@@ -2467,20 +2467,29 @@ orchestration:
           email: demo@demo.com
       ## @extra orchestration.security.initialization.defaultRoles assigning initial users to default roles. More roles can be added to the dictionary: https://docs.camunda.io/docs/next/components/concepts/access-control/authorizations/#default-roles
       ## @param orchestration.security.initialization.defaultRoles.admin.users defines the initial users that will get the admin permission
+      ## @param orchestration.security.initialization.defaultRoles.admin.mappingRules defines the mapping rule IDs for role assignment
       ## @param orchestration.security.initialization.defaultRoles.connectors.users defines the initial users that will get the connectors permission
       ## @param orchestration.security.initialization.defaultRoles.connectors.clients define clients for the orchestration cluster.
+      ## @param orchestration.security.initialization.defaultRoles.connectors.mappingRules defines the mapping rule IDs for role assignment
       defaultRoles:
         admin:
+          mappingRules: []
           users:
             - demo
         connectors:
+          mappingRules: []
           users:
             ## @extra orchestration.security.initialization.defaultRoles.connectors.users[0].connectors Needed for basic auth setup. Can be removed for OIDC. Define the connectors user with the connectors role.
             - connectors
           clients:
             ## @extra orchestration.security.initialization.defaultRoles.connectors.clients[0].connectors Needed for OIDC setup. Can be removed for basic auth. Define the connectors client with the connrectors role.
             - connectors
-      ## @param orchestration.security.initialization.mappingRules define mapping rules.
+      ## @param orchestration.security.initialization.mappingRules define mapping rules when connected to an OIDC provider.
```

Minor comment wording suggestion to respect the conjunctions 
```suggestion
      ## @param orchestration.security.initialization.mappingRules defines the mapping rules when connected to an OIDC provider.
```